mingw-w64-openssl/PKGBUILD

```diff
@@ -36,6 +36,7 @@ validpgpkeys=(
   '7953AC1FBC3DC8B3B292393ED5E9E43F7DF9EE8C' # Richard Levitte <richard@levitte.org>
   'A21FAB74B0088AA361152586B8EF1A6BA9DA2D5C' # Tomáš Mráz <tm@t8m.info>
   'B7C1C14360F353A36862E4D5231C84CDDCC69C45' # Paul Dale <pauli@openssl.org>
+  'EFC0A467D613CB83C7ED6D30D894E2CE8B3D79F5' # OpenSSL Management Committee <openssl-omc@openssl.org>
```

Comment:
I had to add this PGP key from

https://www.openssl.org/source/

> The releases can be also signed by the OpenSSL OMC key with fingerprint [EFC0 A467 D613 CB83 C7ED 6D30 D894 E2CE 8B3D 79F5](https://keys.openpgp.org/search?q=openssl-omc%40openssl.org).

The `get GPG key(s)` step associates it with a different identity, though.

>gpg: key D894E2CE8B3D79F5: public key "OpenSSL security team <openssl-security@openssl.org>" imported

Maybe the comment should contain that identity.

Reply:
I _guess_ this is just a subkey or something (can't verify this right now), so I believe we're good!